### DISCUSSION TITLE
0.15.2

### DISCUSSION THREAD

**Discussion by SchrodingersGat** [inventree/InvenTree#7325] [Announcements]
## Security Advisory

We were recently made aware of a security issue that enables logging in to the API without MFA - even if configured - https://github.com/inventree/InvenTree/security/advisories/GHSA-2crp-q9pc-457j

### Security Patches
* Merge pull request from GHSA-2crp-q9pc-457j by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/7320
### Bug Fixes
* [0.15.x] Fix for email template (#7249) by @github-actions in https://github.com/inventree/InvenTree/pull/7251
* [0.15.x] Add Meta subclass for build serializers (#7315) by @github-actions in https://github.com/inventree/InvenTree/pull/7316
### Devops / Setup Changes
* [0.15.x] fix: SELinux labels for Caddyfile (#7261) by @github-actions in https://github.com/inventree/InvenTree/pull/7262
* [0.15.x] Add clearer error message for invalid SITE_URL (#7311) by @github-actions in https://github.com/inventree/InvenTree/pull/7312
### Other Changes
* Update version.py by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/7252

**Full Changelog**: https://github.com/inventree/InvenTree/compare/0.15.1...0.15.2

<hr /><em>This discussion was created from the release <a href='https://github.com/inventree/InvenTree/releases/tag/0.15.2'>0.15.2</a>.</em>

**Comment by matmair** [comment]
Many thanks to @sur5r who discovered https://github.com/inventree/InvenTree/security/advisories/GHSA-2crp-q9pc-457j
